### ISSUE TITLE
non-existent old VMs displaying in PD Runner

### ISSUE THREAD

**Issue by ddomino007** [lihaoyun6/PD-Runner#142]
<img width="354" alt="image" src="https://user-images.githubusercontent.com/23064460/145557648-f1b5b4ec-3ecd-489a-a590-39602d9512bc.png">
<img width="652" alt="image" src="https://user-images.githubusercontent.com/23064460/145557669-3705995f-b415-4bb6-bb56-89766ae01a98.png">
<img width="682" alt="image" src="https://user-images.githubusercontent.com/23064460/145557735-ef592f2e-a571-4939-b77e-941654e31b03.png">

**Comment by lihaoyun6** [comment]
@ddomino007  

The content of the menu is not updated in real time. PD Runner reads the list of VMs every 5 seconds.  
And if the menu is being displayed, PD Runner will not refresh it. Because doing so will cause the menu to "flicker".  
So the content of the menu may change the next time you expand the menu.

**Comment by ddomino007** [comment]
it did not. I installed PD Runner yesterday and it still didn't change.

**Comment by ddomino007** [comment]
these vms are remnants of a migration from my older Mac. parallels supposedly got rid of them however. nowhere in its indexes does it have these. so my question is where does pd runner look and can I safely clear it.

**Comment by lihaoyun6** [comment]
@ddomino007 
1. Is your Mac based on Intel or Apple Silicon?
2. The version of Parallels Desktop and PD Runner you have installed.
3. Please try to run this command and send me the result (text or screenshot):  
    ```bash
   /usr/local/bin/prlctl list -ao name,-,-,-,-,ostemplate,-,-,-,-,uuid,-,-,-,-,status -s uuid 2>/dev/null|sed 's/[ ]*-[ ]*/-/g;s/----/:/g'
   ```
4. Please try to run this command and send me the `pd.zip` on the desktop: 
    ```bash
    zip -jr ~/Desktop/pd.zip ~/Library/Preferences/com.parallels.Parallels\ Desktop.plist
    ```

**Comment by ddomino007** [comment]
1. My current Mac is an apple silicon Mac (Late 2020 M1 MacBook Air). My previous Mac that I migrated from was a 13 inch late 2013 MacBook Pro (so intel.).
2. Parallels Desktop 17.0.1 (51482) & PD Runner 0.3.2 (32)
3. <img width="682" alt="image" src="https://user-images.githubusercontent.com/23064460/145566574-c3d7eb62-8226-4912-8eb8-9efc264c2de6.png">
4. [pd.zip](https://github.com/lihaoyun6/PD-Runner/files/7692317/pd.zip)

**Comment by lihaoyun6** [comment]
@ddomino007  

<img width="484" alt="Xnip2021-12-10_20-24-02" src="https://user-images.githubusercontent.com/16348097/145573674-ae07474d-4c13-4e9b-b8dc-af7997c05c22.png">

I found these two lines  in the configuration file of Parallels Desktop. And now I know what happened.  
I will fix this problem in the next version of PD Runner. Thank you for your support.

**Comment by ddomino007** [comment]
Gotcha!
Very glad to help and thanks for your assistance ^^

**Comment by lihaoyun6** [comment]
@ddomino007 
This issue has been resolved in the [v0.3.3](https://github.com/lihaoyun6/PD-Runner/releases/tag/0.3.3) update.
